src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -177,6 +177,9 @@ function BaseValidateCodeForm(props) {
      */
     const resendValidateCode = () => {
         User.resendValidateCode(props.credentials.login);
+        setValidateCode('');
+        inputValidateCodeRef.current.clear();
+        inputValidateCodeRef.current.focus();
```

Comment:
Why must we `clear()` here?

But not here:

https://github.com/Expensify/App/blob/cec77d2e583558156b128500da886cedfb1b2a6e/src/pages/settings/Profile/Contacts/ValidateCodeForm/BaseValidateCodeForm.js#L111-L115